openedx/core/djangoapps/call_stack_manager/tests.py

```diff
@@ -162,7 +201,7 @@ def test_parameterized_donottrack(self, log_capt):
         ModelAnotherCallStckMngr(id_field=1).save()
         ModelMixinCallStckMngr(id_field=1).save()
         donottrack_child_func()
-        self.assertEqual(ModelMixinCallStckMngr, log_capt.call_args[0][1])
+        self.assertEqual(ModelMixinCallStckMngr, log_capt.call_args[0][2])
```

Comment:
This is very hard to read. It's unclear what's being asserted, because of the double nesting of indexes into log_capt.